docs/topics/templates.txt

```diff
@@ -611,6 +611,49 @@ creating an object that specifies the following attributes:
 
 .. currentmodule:: django.template
 
+.. _overriding-templates:
+
+Overriding templates
+====================
+
+To override templates from a package in ``django.contrib`` or from a third-party
+package, the preferred method is to create replacement templates in your
+project's templates directory.
+
+Let's say you're trying to override the templates for a third-party package
+called ``blog``, which provides the templates ``post.html`` and ``list.html``.
+The relevant settings for your project would look like::
+
+    INSTALLED_APPS = [
+        ...,
+        blog,
+        ...,
+    ]
+
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': ['templates'],
```

Comment:
Does this actually work reliably? I thought you needed to do `os.path.join(BASE_DIR, 'templates')`